hardhat/task-test-get-files.js

```diff
@@ -0,0 +1,36 @@
+// Ignores the proxy tests after they are removed by the transpiler
+
+const { internalTask } = require('hardhat/config');
+const { TASK_TEST_GET_TEST_FILES } = require('hardhat/builtin-tasks/task-names');
+
+internalTask(TASK_TEST_GET_TEST_FILES).setAction(async ({ testFiles }, { config }) => {
```

Comment:
same, we need some documentation here

Reply:
There was a comment at the top of the file. But I moved it down and rewrote it a little.